### PR TITLE
Omi mmt login

### PR DESCRIPTION
**Pull request by onkar-projects**

### CODE REVIEW

File: src/main/java/com/pom/LoginPage.java [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+	@FindBy(xpath = "//input[@class='font14 fullWidth']")
+	private WebElement username;
+	
+		@FindBy(xpath = "//section[@class='modalMain tcnFooter']//span[@class='commonModal__close']")
+		private WebElement closeLoginpg;
+	
+	@FindBy(xpath = "//button[@class='capText font16']")
+	private WebElement contiueBtn;
+	
+
```
Comment (satyashripisal22) [inline src/main/java/com/pom/LoginPage.java]: remove the spaces

File: src/main/java/com/utility/BaseClass.java [hunk 2]
```diff
@@ -24,9 +35,8 @@ public WebDriver IntializationBrowser() {
 		return driver;
```
Comment (satyashripisal22) [inline src/main/java/com/utility/BaseClass.java]: driver intialization code

File: test-output/Default suite/Default test.html [hunk 3]
```diff
@@ -55,17 +55,31 @@
 <body>
 <h2 align='center'>Default test</h2><table border='1' align="center">
```
Comment (satyashripisal22) [inline test-output/Default suite/Default test.html]: remove this Default Test file

File: target/classes/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -1,4 +1,4 @@
 Manifest-Version: 1.0
```
Comment (satyashripisal22) [inline target/classes/META-INF/MANIFEST.MF]: remove the target class

File: test-output/Default suite/Default test.xml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/Default suite/Default test.xml]: remove test output

File: test-output/index.html [hunk 6]
```diff
@@ -50,7 +50,7 @@
               <ul>
```
Comment (satyashripisal22) [inline test-output/index.html]: remove index.html

File: test-output/junitreports/TEST-com.test.LoginTest.xml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/junitreports/TEST-com.test.LoginTest.xml]: removed junitreport

File: test-output/testng-failed.xml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/testng-failed.xml]: remove test-failed.xml

File: test-output/testng-results.xml [hunk 9]
```diff
@@ -1,11 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/testng-results.xml]: remove testng-result.xml

**Review by satyashripisal22 — CHANGES_REQUESTED** [review]
make changes based on review

File: src/main/java/com/pom/LoginPage.java (hunk 1 above)
Comment (onkar-projects) [reply src/main/java/com/pom/LoginPage.java]: I done my changes 
please review it agian

File: target/classes/META-INF/maven/MakeMyTrip/MakeMyTrip/pom.properties [hunk 10]
```diff
@@ -1,7 +1,7 @@
 #Generated by Maven Integration for Eclipse
```
Comment (satyashripisal22) [inline target/classes/META-INF/maven/MakeMyTrip/MakeMyTrip/pom.properties]: deleted this as well

File: test-output/Default suite/Default test.xml [hunk 11]
```diff
@@ -1,4 +1,5 @@
...
 <!-- Generated by org.testng.reporters.JUnitXMLReporter -->
```
Comment (satyashripisal22) [inline test-output/Default suite/Default test.xml]: delete this

File: test-output/Default suite/testng-failed.xml [hunk 12]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/Default suite/testng-failed.xml]: delete this

File: test-output/emailable-report.html [hunk 13]
```diff
@@ -9,9 +9,10 @@
 <table>
```
Comment (satyashripisal22) [inline test-output/emailable-report.html]: delete this

File: test-output/junitreports/TEST-com.test.LoginTest.xml [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+<!-- Generated by org.testng.reporters.JUnitReportReporter -->
```
Comment (satyashripisal22) [inline test-output/junitreports/TEST-com.test.LoginTest.xml]: delete

File: test-output/junitreports/TEST-com.test.SearchPageTest.xml [hunk 15]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (satyashripisal22) [inline test-output/junitreports/TEST-com.test.SearchPageTest.xml]: delete

File: test-output/testng-failed.xml (hunk 8 above)
Comment (satyashripisal22) [inline test-output/testng-failed.xml]: delete

**Review by satyashripisal22 — CHANGES_REQUESTED** [review]

File: src/main/java/com/utility/BaseClass.java (hunk 2 above)
Comment (onkar-projects) [reply src/main/java/com/utility/BaseClass.java]: code it there but it not showing in the changes but it is still there

File: test-output/Default suite/Default test.html (hunk 3 above)
Comment (onkar-projects) [reply test-output/Default suite/Default test.html]: this file is removed

File: target/classes/META-INF/MANIFEST.MF (hunk 4 above)
Comment (onkar-projects) [reply target/classes/META-INF/MANIFEST.MF]: this file is removed

File: test-output/Default suite/Default test.xml (hunk 5 above)
Comment (onkar-projects) [reply test-output/Default suite/Default test.xml]: this file it deleted

File: test-output/index.html (hunk 6 above)
Comment (onkar-projects) [reply test-output/index.html]: this file is removed

File: test-output/junitreports/TEST-com.test.LoginTest.xml (hunk 7 above)
Comment (onkar-projects) [reply test-output/junitreports/TEST-com.test.LoginTest.xml]: this file is removed

File: test-output/testng-results.xml (hunk 9 above)
Comment (onkar-projects) [reply test-output/testng-results.xml]: this file is removed

File: target/classes/META-INF/maven/MakeMyTrip/MakeMyTrip/pom.properties (hunk 10 above)
Comment (onkar-projects) [reply target/classes/META-INF/maven/MakeMyTrip/MakeMyTrip/pom.properties]: this file is deleted

File: test-output/Default suite/Default test.xml (hunk 11 above)
Comment (onkar-projects) [reply test-output/Default suite/Default test.xml]: this file is deleted

File: test-output/Default suite/testng-failed.xml (hunk 12 above)
Comment (onkar-projects) [reply test-output/Default suite/testng-failed.xml]: this file is deleted

File: test-output/emailable-report.html (hunk 13 above)
Comment (onkar-projects) [reply test-output/emailable-report.html]: this file  is deleted

File: test-output/junitreports/TEST-com.test.LoginTest.xml (hunk 14 above)
Comment (onkar-projects) [reply test-output/junitreports/TEST-com.test.LoginTest.xml]: this file is deleted

File: test-output/junitreports/TEST-com.test.SearchPageTest.xml (hunk 15 above)
Comment (onkar-projects) [reply test-output/junitreports/TEST-com.test.SearchPageTest.xml]: this file is deleted

File: test-output/testng-failed.xml (hunk 8 above)
Comment (onkar-projects) [reply test-output/testng-failed.xml]: this file is deleted

**Review by satyashripisal22 — APPROVED** [review]

**Pull request merged**